client/functions.lua

```diff
@@ -177,29 +178,36 @@ function QBCore.Functions.GetPlayersFromCoords(coords, distance)
     return closePlayers
 end
 
-function QBCore.Functions.GetClosestVehicle()
-    local ped = PlayerPedId()
-    local coords = GetEntityCoords(ped)
-    local vehicles = GetGamePool('CVehicle')
-    local closestDistance = -1
-    local closestVehicle = -1
-    for i = 1, #vehicles, 1 do
-        local vehicleCoords = GetEntityCoords(vehicles[i])
-        local distance = #(vehicleCoords - coords)
-        if closestDistance == -1 or closestDistance > distance then
-            closestVehicle = vehicles[i]
-            closestDistance = distance
-        end
-    end
-    return closestVehicle, closestDistance
+function QBCore.Functions.GetClosestVehicle(coords)
+	local ped = PlayerPedId()
+	local vehicles        = GetGamePool('CVehicle')
```

Comment:
Would probably be a good idea to keep the code consistency with how this was before, this doesn't really improve readability at all